lib/dead_code_detector/initializer.rb

```diff
@@ -34,13 +37,18 @@ def enable(klass)
       end
 
       def enable_for_cached_classes!
-        return if @enabled
+        return if fully_enabled
         return unless allowed?
-        @enabled = true
-        cached_classes.each do |class_name|
+        classes = cached_classes.sort.to_a
```

Comment:
Question: could the `cached_classes` array being changed (for example, some classes are removed in redis) between running this batch enable method `enable_for_cached_classes!`?

Reply:
@tedyangx I think it is possible if a concurrent "refresh" happened. However, I believe that would be acceptable. 

When things are refreshed, in order for the refresh methods to be picked up, we'd need the processes to be re-started. Once we have restarted, we will iterate over the cache classes from the start. So I believe that is okay.